### PR TITLE
Add RFI link

### LINKED ISSUE #1321: Add RFI link (once it becomes available)
**Describe the bug**
Link to federalregister.gov/whatever the link is to link to the CEJST RFI.

**To Reproduce**
1. Go to 'Explore the tool'
2. Scroll down to 'links below the map'

**Expected behavior**
Once the RFI is available, we need to link to it.

**Screenshots**
Screen Shot 2022-02-22 at 9 32 29 AM

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Additional context**

### PR DESCRIPTION
**Pull request by vim-usds**
closes #1321

### CODE REVIEW

**Comment by KameronKerger** [comment]
QA passed

File: client/src/components/HowYouCanHelp/tests/__snapshots__/howYouCanHelp.test.tsx.snap [hunk 1]
```diff
@@ -30,7 +30,17 @@ exports[`rendering of the HowYouCanHelp checks if various text fields are visibl
         .
       </li>
       <li>
-        CEQ will publish a Request for Information on federalregister.gov.
+        CEQ will publish a Request for Information on
```
Comment (lucasmbrown-usds) [inline client/src/components/HowYouCanHelp/tests/__snapshots__/howYouCanHelp.test.tsx.snap:33]: Nit: @KameronKerger is this the right language to use future tense here? Would it be more like `CEQ has published a Request...`

**Review by lucasmbrown-usds — APPROVED** [review]
Asked about language but otherwise LGTM!

**Comment by KameronKerger** [comment]
yes, good nit @lucasmbrown-usds. @vim-usds we should change it to 
CEQ **has published** a Request for Information on federalregister.gov

**Pull request merged**